src/media/js/main.js

```diff
@@ -230,6 +202,8 @@ function(_) {
             $('#site-nav').after(nunjucks.env.render('incompatible.html'));
         }
 
+        update_banner.showIfneeded();
```

Comment:
We could. I wanted to be sure it happens after everything, including header and footer, have been rendered properly.